src/main/java/com/imsweb/mph/HematoUtils.java

```diff
@@ -9,37 +9,34 @@ public final class HematoUtils {
 
     private static Pattern _MORPHOLOGY = Pattern.compile("^(\\d{4}/\\d)");
 
-    public static boolean isSamePrimary(String morph1, String morph2, int year1, int year2) {
```

Comment:
I really don't love these changes.   You are not requiring everyone to provide a provider to each of these methods. 

But except for SEER*API, no software cares or use a provider, they all use the default one. So now they will need to get the default provider for these calls? But not to use MphUtils itself? That seems all wrong.

I would remove this utility class and move these methods to MphUtils, and make them non-static and use the provider from that class. 

Why having multiple entry points for such a simple library; and if these methods need a provider, putting them in MphUtils, which has a provider already, makes much more sense IMO.